<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save_As"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menu_Brush_Styles_2">
     <property name="title">
      <string>&amp;Brush Styles</string>
     </property>
     <addaction name="actionDence_1"/>
     <addaction name="actionDence_2"/>
     <addaction name="actionHor"/>
     <addaction name="actionVertical"/>
     <addaction name="actionNone_Brush"/>
     <addaction name="actionSolid_2"/>
    </widget>
    <widget class="QMenu" name="menu_Pen_Style">
     <property name="title">
      <string>&amp;Pen Style</string>
     </property>
     <addaction name="actionSolid"/>
     <addaction name="actionDash"/>
     <addaction name="actionDot"/>
     <addaction name="actionDash_Dot"/>
     <addaction name="actionDash_Dot_Dot"/>
    </widget>
    <addaction name="action_Clear"/>
    <addaction name="menu_Brush_Styles_2"/>
    <addaction name="menu_Pen_Style"/>
    <addaction name="separator"/>
    <addaction name="actionWidth"/>
    <addaction name="action_Text_Size"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Figures</string>
    </property>
    <addaction name="actionRectangle"/>
    <addaction name="actionEllipse"/>
    <addaction name="actionTriangle"/>
    <addaction name="actionRhombus"/>
    <addaction name="actionHexagon"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPen"/>
   <addaction name="actionBrush"/>
   <addaction name="actionFill"/>
   <addaction name="actionText"/>
   <addaction name="action_Color"/>
   <addaction name="action_Print"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/img/print.png</normaloff>:/res/img/print.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
  </action>
  <action name="action_Color">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/img/color.jpg</normaloff>:/res/img/color.jpg</iconset>
   </property>
   <property name="text">
    <string>&amp;Color</string>
   </property>
  </action>
  <action name="action_Clear">
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
  <action name="actionPen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/img/pen.png</normaloff>:/res/img/pen.png</iconset>
   </property>
   <property name="text">
    <string>Pen</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/img/fill.png</normaloff>:/res/img/fill.png</iconset>
   </property>
   <property name="text">
    <string>Fill</string>
   </property>
  </action>
  <action name="actionLinear_Gradient">
   <property name="text">
    <string>Linear Gradient</string>
   </property>
  </action>
  <action name="actionHor">
   <property name="text">
    <string>Horizontal</string>
   </property>
  </action>
  <action name="actionDence_1">
   <property name="text">
    <string>Dence 1</string>
   </property>
  </action>
  <action name="actionDence_2">
   <property name="text">
    <string>Dence 2</string>
   </property>
  </action>
  <action name="actionVertical">
   <property name="text">
    <string>Vertical</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionTriangle">
   <property name="text">
    <string>Triangle</string>
   </property>
  </action>
  <action name="actionRhombus">
   <property name="text">
    <string>Rhombus</string>
   </property>
  </action>
  <action name="actionHexagon">
   <property name="text">
    <string>Hexagon</string>
   </property>
  </action>
  <action name="actionText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/img/text.png</normaloff>:/res/img/text.png</iconset>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </action>
  <action name="actionBrush">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/img/brush.png</normaloff>:/res/img/brush.png</iconset>
   </property>
   <property name="text">
    <string>Brush</string>
   </property>
  </action>
  <action name="actionSolid">
   <property name="text">
    <string>Solid</string>
   </property>
  </action>
  <action name="actionDash">
   <property name="text">
    <string>Dash</string>
   </property>
  </action>
  <action name="actionDot">
   <property name="text">
    <string>Dot</string>
   </property>
  </action>
  <action name="actionDash_Dot">
   <property name="text">
    <string>Dash Dot</string>
   </property>
  </action>
  <action name="actionDash_Dot_Dot">
   <property name="text">
    <string>Dash Dot Dot</string>
   </property>
  </action>
  <action name="actionWidth">
   <property name="text">
    <string>&amp;Pen Width</string>
   </property>
  </action>
  <action name="action_Text_Size">
   <property name="text">
    <string>&amp;Text Size</string>
   </property>
  </action>
  <action name="actionNone_Brush">
   <property name="text">
    <string>None Brush</string>
   </property>
  </action>
  <action name="actionSolid_2">
   <property name="text">
    <string>Solid</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
